api/v1/swagger/paths/holdings.json

```diff
@@ -0,0 +1,352 @@
+{
+  "/holdings": {
+    "get": {
+      "x-mojo-to": "Holding#list",
+      "operationId": "listHoldings",
+      "tags": ["patrons", "holdings"],
+      "parameters": [
+        {
+          "name": "holding_id",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal holding identifier",
+          "type": "integer"
+        },
+        {
+          "name": "biblionumber",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio identifier",
+          "type": "integer"
+        },
+        {
+          "name": "biblioitemnumber",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio item identifier",
+          "type": "integer"
+        },
+        {
+          "name": "frameworkcode",
+          "in": "query",
+          "description": "Case insensitive starts-with search for framework used in cataloging this record",
+          "type": "string"
+        },
+        {
+          "name": "holdingbranch",
+          "in": "query",
+          "description": "Case insensitive starts-with search for internal biblio identifier for library that is currently in possession of this holding",
+          "type": "string"
+        },
+        {
+          "name": "location",
+          "in": "query",
+          "description": "Case insensitive starts-with search for authorized value for the shelving location for this holding",
+          "type": "string"
+        },
+        {
+          "name": "ccode",
+          "in": "query",
+          "description": "Case insensitive starts-with search for authorized value for the collection code for this holding",
```

Comment:
You could add a 'ccode_lookup' -field to the endpoint definition to enable "starts-with"-matching, or some other variety of lookup algorithm, such as "*value*", Levenshtein, etc.
By default one would get the exact matches.

Reply:
Removed starts-with matching completely. Default is now an exact match.